external-crates/move/crates/move-compiler/tests/move_check/parser/copy_move_path.exp

```diff
@@ -0,0 +1,16 @@
+error[E13001]: feature is not supported in specified edition
+   ┌─ tests/move_check/parser/copy_move_path.move:12:9
+   │
+12 │         copy x.y.z;
+   │         ^^^^ 'move' and 'copy' on paths is not supported by current edition 'legacy', only '2024.alpha' support this feature
```

Comment:
"on paths" -> "for paths"?

Reply:
I ended up reusing the Move2024Paths feature, let me know though if that error is too vague for this